package.json

```diff
@@ -13,7 +13,7 @@
     "build-packages": "preconstruct build",
     "build": "./scripts/build.sh",
     "vercel-build": "echo \"you're welcome - nothing to build\"",
-    "generate-icons": "yarn workspaces foreach -p --include @commercetools-website/docs-smoke-test,@commercetools-docs/ui-kit run generate-icons && prettier --write '**/generated/*.{js,ts,tsx}'",
```

Comment:
Using `workspaces foreach` has the downside of suppressing errors, which was the case here (the smoke tests command was failing to run).